specification/maps/data-plane/Search/preview/2023-04-01-preview/search.json

```diff
@@ -694,17 +710,6 @@
                   "type": "string",
```

Comment:
Can we also remove `Optional.` in line 692 for `countryRegionSet`? Same reason as `circularMapView`